index.html

```diff
@@ -299,6 +299,35 @@ <h2>Contact me</h2>
     <hr class="contact-hr">
   </section>
 </section>
+
+<div class="modal-container">
+  <div class="modalbox">
+    <div class="pop-top">
+      <div class="top-title">
+        <h3 class="modaltitle"></h3>
+        <img onclick="CloseModal()" class="close-modal" src="images/close.jpg" alt="closebutton">
+    </div>
+    <div class="project_info">
+        <p class="pop-company"></p>
+      <ul class="ul">
+        <li class="pop-role"></li>
+        <li class="pop-year"></li>
+    </ul>
+    </div>
+  </div>
+    <img class="popup-image" alt="Project Popup Image">
+    <div class="below-box">
+    <p class="pop-up-description"></p>
+    <div class="sec-box">
+      <ul class="language_used_modal"></ul>
+    <div class="modal-button-div">  
+  <a  class="seelive"><button type="button">See live <img class="button-image" src="images/export.svg" alt="live"> </button></a>    
+  <a  class="seesource"><button type="button">See source  <img class="button-image" src="images/github.png" alt="github"></button></a>  
```

Comment:
- [ ] These two buttons should be aligned in a horizontal format like in the prvided desgin, kindly fix this.

Figma design
![image](https://user-images.githubusercontent.com/81932478/202532154-617126a1-b434-45b9-a0e6-9ece7a437235.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/202532307-b96fb997-cf62-45f9-99f9-b25a48ca44c2.png)

- [ ] Kindly lign the buttons to match the given design for the mobile view as well. The buttons should span the full width of the image above.

Given design
![image](https://user-images.githubusercontent.com/81932478/202532885-f899f475-dc9d-4fb6-b1f2-f4210dd5102e.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/202533001-9c033f69-a4e1-4bb1-97c2-08cf5cd4c873.png)